tests/test_prob.py

```diff
@@ -66,6 +66,22 @@ def circuit():
     expected = np.einsum("ijkl->jl", expected).flatten()
     assert np.allclose(res, expected, atol=tol, rtol=0)
 
+def test_marginal_prob_more_wires(init_state, tol):
```

Comment:
You could also include `mocker` here, and have
```python
spy = mocker.spy(dev, "states_to_binary")
...
spy.assert_called_once()
```
:)

Reply:
Added, thanks!